Mathlib/Data/Nat/Bitwise.lean

```diff
@@ -18,7 +18,7 @@ import Mathlib.Tactic.Ring
 
 In the first half of this file, we provide theorems for reasoning about natural numbers from their
 bitwise properties. In the second half of this file, we show properties of the bitwise operations
-`lor'`, `land'` and `lxor'`, which are defined in core.
+`lor`, `land` and `xor'`, which are defined in core.
```

Comment:
I assume you meant to drop the prime too:
```suggestion
`lor`, `land` and `xor`, which are defined in core.
```

Reply:
Yes I did. These primes are easy to overlook, thanks for double-checking